### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by Zerowhizkey**
Implemented loader function to profile/search

### CODE REVIEW

File: src/routes/Search/Search.tsx [hunk 1]
```diff
@@ -48,6 +55,10 @@ const Search = () => {
         }
     }, [selectedGuest]);
 
+    // if (loading) {
```
Comment (linda-94-andersson) [inline src/routes/Search/Search.tsx:58]: Please remove code that is not used! :)

File: src/routes/Search/Search.tsx [hunk 2]
```diff
@@ -48,6 +55,10 @@ const Search = () => {
...
+    //     return <LoadingOverlay />;
```
Comment (linda-94-andersson) [inline src/routes/Search/Search.tsx:59]: Should this stay?

**Review by linda-94-andersson — CHANGES_REQUESTED** [review]
Please see if the out-commented code should be there or not.

File: src/routes/Search/Search.tsx (hunk 1 above)
Comment (Zerowhizkey) [reply src/routes/Search/Search.tsx]: oh sheit im fiximg

**Review by linda-94-andersson — APPROVED** [review]
Well done!

**Pull request merged**